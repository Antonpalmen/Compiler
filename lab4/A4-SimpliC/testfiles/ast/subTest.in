int a() {
    int x;
    x = 7 - 5;
}
