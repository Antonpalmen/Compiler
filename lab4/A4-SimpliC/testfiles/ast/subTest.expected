Program
  List
    Function
      IdDecl ID="a"
      List
      List
        Declaration
          IdDecl ID="x"
          Opt
        Assignment
          IdDecl ID="x"
          Sub
            Numeral NUMERAL="7"
            Numeral NUMERAL="5"
